addon/components/img-lazy.js

```diff
@@ -41,6 +41,14 @@ export default Component.extend({
 
     return placeholder(get(this, 'width'), get(this, 'height'))
   }),
+
+  _srcset: computed('srcset', function() {
+    if ((isFastBoot && !get(this, 'lazyFastBoot')) ||
+      (!isFastBoot && !get(this, 'observer.hasIntersectionObserver'))) {
```

Comment:
We could deduplicate this into a separate method/property.

Reply:
LOL I can remember that I thought about it but forgot to do this.